nucypher/cli/commands/felix.py

```diff
@@ -129,16 +127,11 @@ def create_character(self, emitter, config_file, debug):
                                     character_configuration=felix_config,
                                     password=get_nucypher_password(confirm=False))
 
-            # Produce Teacher Ursulas
-            teacher_nodes = actions.load_seednodes(emitter,
-                                                   teacher_uris=self.teacher_uris,
-                                                   min_stake=self.min_stake,
-                                                   federated_only=felix_config.federated_only,
-                                                   network_domains=felix_config.domains,
-                                                   network_middleware=self.middleware)
+            client_password = get_client_password(checksum_address=felix_config.checksum_address,
+                                                  envvar="NUCYPHER_WORKER_ETH_PASSWORD")
```

Comment:
Now we have a constant for the name of this envvar in `nucypher.config.constants` called `NUCYPHER_ENVVAR_WORKER_ETH_PASSWORD`, in case you want to use it.

Reply:
:heavy_check_mark: